### PR TITLE
Catch `ValueError` regardless of py version.

### PR DESCRIPTION
**Pull request by mpapierski**
A small adhoc change. 

It turns out on travis sometimes we run rita.py on py2, where JSONDecodeError doesn't exist.

Example failed job: https://travis-ci.org/althea-mesh/althea_rs/jobs/413146414

### CODE REVIEW

File: integration-tests/rita.py [hunk 1]
```diff
@@ -693,7 +693,9 @@ def test_endpoints_all(self):
                     pprint(neighbors)
                 else:
                     print(result.stdout.read().decode('utf-8'))
-            except json.JSONDecodeError as e:
+            except ValueError as e:
+                if VERBOSE:
+                    print('Unable to decode JSON {!r}: {!r}'.format(result, e))
```
Comment (drozdziak1) [inline integration-tests/rita.py]: Unnecessary, the `assert_test` below will print `<red plus> Decoding the JSON Failed`

File: integration-tests/rita.py [hunk 2]
```diff
@@ -711,7 +713,9 @@ def test_endpoints_all(self):
                     pprint(exits)
                 else:
                     print(result.stdout.read().decode('utf-8'))
-            except json.JSONDecodeError as e:
+            except ValueError as e:
+                if VERBOSE:
+                    print('Unable to decode JSON {!r}: {!r}'.format(result, e))
```
Comment (drozdziak1) [inline integration-tests/rita.py]: same here

File: integration-tests/rita.py [hunk 3]
```diff
@@ -729,7 +733,9 @@ def test_endpoints_all(self):
                     pprint(info)
                 else:
                     print(result.stdout.read().decode('utf-8'))
-            except json.JSONDecodeError as e:
+            except ValueError as e:
+                if VERBOSE:
+                    print('Unable to decode JSON {!r}: {!r}'.format(result, e))
```
Comment (drozdziak1) [inline integration-tests/rita.py]: here

File: integration-tests/rita.py [hunk 4]
```diff
@@ -747,7 +753,9 @@ def test_endpoints_all(self):
                     pprint(settings)
                 else:
                     print(result.stdout.read().decode('utf-8'))
-            except json.JSONDecodeError as e:
+            except ValueError as e:
+                if VERBOSE:
+                    print('Unable to decode JSON {!r}: {!r}'.format(result, e))
```
Comment (drozdziak1) [inline integration-tests/rita.py]: and here

**Comment by mpapierski** [comment]
@drozdziak1 As discussed I removed `if VERBOSE` branch and it will report invalid responses unconditionally. Updated with some more changes because because I realized I'll be reporting `Popen` object instead of actual `stdout` output.

**Pull request merged**